CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.1.0
+
+- Add Function Calling functionality to `chat` and `chatStream`
```

Comment:
It might be worth to add the info about supporting `response_format` as well 🤔

Reply:
Updated in https://github.com/nomtek/mistralai_client_dart/pull/39/commits/34fba38d832abe889096e4563a06391ea18a7a19